scripts/run-two-instances.sh

```
@@ -0,0 +1,103 @@
+docker exec -d horcrux-steward bash -c "\
+    source /workspace/scripts/setup-x11.sh && \
+    cd /workspace && \
+    flutter run -d linux --debug --host-vmservice-port=9181 --no-dds \
+    &> /tmp/flutter_run.log"
+
+echo ""
+echo -e "${BLUE}Waiting for apps to start (this takes ~2-3 minutes)...${NC}"
+echo ""
+
+# Wait for VM service URIs
+for i in $(seq 1 36); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)